site/en/docs/extensions/reference/tabs/index.md

```diff
@@ -24,11 +24,57 @@ permission in the manifest, as shown below:
 
 ## Examples
 
+Opening the extension's onboarding page in new tab after the extension is installed for the first
+time. Must be called from a background context.
+
+```js
+chrome.runtime.onInstalled.addListener((reason) => {
+  if (reason === chrome.runtime.OnInstalledReason.INSTALL) {
+    chrome.tabs.create({
+      url: 'onboarding.html'
+    });
+  }
+});
+```
+
+Retrieve the user's currently focused tab. Requires Manifest V3. Cannot be used in a content
```

Comment:
Why does it need V3? Purely because it uses `Promise`?

Reply:
Yep, exactly. TBH I'm a bit conflicted here. On one hand I want to provide snippets that are as broadly useful as possible, on the other I want to encourage modern language features. Perhaps it's enough if I just clarify this line to say "Requires Manifest V3 due to use of promises"?

Reply:
I should note that I also considered displaying both a promise & callback version of this snippet, but decided against it because it took up too much vertical space. Another use case in support of #90.

Reply:
We talked about this; we want this to be a going forward document, so I think `Promise` only (with a note) makes sense.

Over the next year I presume we'll stop including the note as much.

Reply:
Yep, I think that makes sense.